Description: Update doxygen header file to compile
 This patch correct a bug in doxygen when using custom header
 including '\+' char in various places in the generated LaTeX
 files. This patch declare the '\+' LaTeX command as an empty
 command to avoid compilation failure.
Author: Philippe Thierry
Last-Update: 2017-04-28
--- a/src/tex/header.tex
+++ b/src/tex/header.tex
@@ -106,6 +106,9 @@
   \newpage{\pagestyle{empty}\cleardoublepage}%
 }
 
+% correcting doxygen bug due to custom header
+\newcommand{\+}{}
+
 
 %===== C O N T E N T S =====
 
